### PR TITLE
9485 GitHub organizations

### LINKED ISSUE #9485: Github oauth: organization signup

### PR DESCRIPTION
**Pull request by javitonino**
Closes #9485

### CODE REVIEW

**Comment by javitonino** [comment]
Please CR @gfiorav

File: app/controllers/carto/github_controller.rb [hunk 1]
```diff
@@ -18,11 +18,11 @@ def github
       api = Github::Api.with_code(@github_config, code)
 
       user = login(api)
-      unless user
-        return redirect_to CartoDB.url(self, 'login')
+      if user
+        redirect_to user.public_url << CartoDB.path(self, 'dashboard', trailing_slash: true)
```
Comment (gfiorav) [inline app/controllers/carto/github_controller.rb]: 😨

**Comment by gfiorav** [comment]
📦 , 📦 , 📦

**Comment by javitonino** [comment]
ping @gfiorav (added a migration to pass github_user_id to the creation queue)

**Comment by gfiorav** [comment]
Ok, 📦 it!

**Pull request merged**